_config.yaml

```diff
@@ -31,9 +31,12 @@ theme: minima
 header_pages:
   # Note that when first cloned, the `workshop` directory will not exist,
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
+  - workshop/HOME.md
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
```

Comment:
These should be accompanied by changes to the frontmatter for these files to add a `nav_title` variable.

Reply:
So for example we could use
```
title: Resources for Consultation Sessions
nav_title: Consultation Sessions
```
?

Reply:
Correct.